### PR TITLE
Add OpenET/ensemble/conus/gridmet/monthly/v2_0

### PR DESCRIPTION
**Pull request by guzman2319**
I'm getting one error I don't understand that says: 'keywords_tree', 'Only one instance of "openet"'.  That seems like a useful keyword for this dataset so hopefully there is some way of getting around this.

### CODE REVIEW

**Comment by simonff** [comment]
> 

I agree - our current system is too restrictive about such keywords. Let me see if I can quickly fix it, but for now please exclude this keyword.

**Comment by guzman2319** [comment]
I removed the openet keyword.  We were also wondering if the public facing Collection ID case has to match the internal one?

**Comment by simonff** [comment]
They don't have to match - we'll copy the assets from the private collection to the public one.

File: catalog/OpenET/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  description: |||
+    The OpenET dataset includes satellite-based data on the total amount of water that
+    is transferred from the land surface to the atmosphere through the process of
+    evapotranspiration (ET). OpenET provides ET data from multiple satellite-driven models,
+    and also calculates a single "ensemble value" from the model ensemble. The models
+    currently included in the OpenET model ensemble are ALEXI/DisALEXI, eeMETRIC, geeSEBAL,
+    PT-JPL, SIMS, and SSEBop. The OpenET ensemble ET value is calculated as the mean of the
+    ensemble after filtering and removing outliers using the median absolute
+    deviation approach.  
+    All models currently use Landsat satellite data to produce ET data at a pixel size of
```
Comment (schwehr) [inline catalog/OpenET/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet:35]: Please insert a blank line to make a new paragraph.  Or if you meant it to still be a single paragraph, please reflow the paragraph.

File: catalog/OpenET/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+      },
+      {
+        name: 'et_ensemble_mad_count',
+        description: |||
+           The number of models used to compute the ensemble ET value,
+           after filtering for outliers using the median absolute deviation (mad)
+        |||,
+      },
+      {
+        name: 'et_ensemble_mad_index',
```
[the remark this answers is not in the document]
Comment (guzman2319) [reply catalog/OpenET/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet:89]: We added the bitmask. Let me know if that's what you were looking for.

**Comment by guzman2319** [comment]
> We _just_ moved our JS code examples to this repo as well: https://github.com/google/earthengine-catalog/tree/main/examples
> 
> Could you please add one?

I added the sample script to the examples directory.

**Comment by guzman2319** [comment]
The PR got closed by mistake. Reopening...

**Review by guzman2319 — COMMENTED** [review]
Added the js example

**Review by simonff — COMMENTED** [review]
Please add asset property descriptions in the gee:schema section. Example: https://github.com/google/earthengine-catalog/blob/main/catalog/GRIDMET/GRIDMET_DROUGHT.jsonnet#L112

**Comment by guzman2319** [comment]
> We _just_ moved our JS code examples to this repo as well: https://github.com/google/earthengine-catalog/tree/main/examples
> 
> Could you please add one?

We added the js code to the examples directory.

File: examples/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// TODO: Update the image collection ID with the public data catalog ID
+var dataset = ee.ImageCollection('projects/openet/ensemble/conus/gridmet/monthly/v2_0')
```
Comment (simonff) [inline examples/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.js:2]: please update to OpenET/ENSEMBLE/CONUS/GRIDMET/MONTHLY/v2_0

File: catalog/OpenET/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
```
Comment (simonff) [inline catalog/OpenET/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet:15]: please add at the top level:
'gee:skip_indexing': true,
'gee:user_uploaded': true,

These are temporary flags the process currently requires.

**Review by simonff — APPROVED** [review]
Great start for 2023!

**Comment by schwehr** [comment]
@guzman2319 : I am finishing up the STAC entry, but have a question:

What are the pixel sizes for the bands in meters?

**Comment by schwehr** [comment]
> @guzman2319 : I am finishing up the STAC entry, but have a question:
> 
> What are the pixel sizes for the bands in meters?

I see 30 m in the description

**Comment by schwehr** [comment]
Thank you for your addition to the Earth Engine Public Data Catalog! 

These changes were committed to main in the https://github.com/google/earthengine-catalog/commit/847ffbcacf85e89fdea5562450d267ecf3e36bb1 merge from pr #305

If you have changes to this catalog entry, please open a new pull request.

By the way, if you want to see your changes in the git history, you can sync to upstream main and do something like this (the oneline is optional)

```
git log --pretty=oneline --author=aguzman@csumb.edu
b8847248c08705aa6b4d38d5aa60cdc53d5e2d68 Create OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.js
8493fe23fb6c8882ff1feb4759c77b1bed23c9db Update OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet
3b8f977a8743bf1dea0086ca96050c494b6dc01f Adding the bitmask description
4a25cf45b2b49acdd8944ab81c4bd8827f013b7b Update OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet
904ae8258e1c2f24d8d026df5e3681fd3ab2ae43 Update OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet
fbb584e7d34cc97bb0c270fb1378c9db4794e19d Update OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet
```

File: examples/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.js (hunk 3 above)
Comment (schwehr) [reply examples/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.js:2]: I did this in #305, so it's all set.

File: catalog/OpenET/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet (hunk 4 above)
Comment (schwehr) [reply catalog/OpenET/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet:15]: I did this in #305, so it's all set.

**Pull request closed without merging**